x11-wm/pekwm/pekwm-0.4.1.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake desktop optfeature
+
+DESCRIPTION="A lightweight window manager initially based on aewm++"
+HOMEPAGE="
+	https://www.pekwm.se/
+	https://github.com/pekwm/pekwm
+"
+SRC_URI="
+	https://github.com/pekwm/pekwm/archive/refs/tags/release-${PV}.tar.gz -> pekwm-${PV}.tar.gz
+"
+S="${WORKDIR}/pekwm-release-${PV}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+IUSE="debug +jpeg pango +png test truetype xinerama +xpm"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	x11-libs/libX11
+	x11-libs/libXext
+	jpeg? ( media-libs/libjpeg-turbo )
+	pango? ( x11-libs/pango x11-libs/cairo[X] )
+	png? ( media-libs/libpng:0 )
+	truetype? ( x11-libs/libXft )
+	xinerama? ( x11-libs/libXinerama )
+	xpm? ( x11-libs/libXpm )"
+
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+src_configure() {
+	local mycmakeargs=(
+		-DENABLE_IMAGE_JPEG=$(usex jpeg)
+		-DENABLE_IMAGE_PNG=$(usex png)
+		-DENABLE_XINERAMA=$(usex xinerama)
+		-DENABLE_XFT=$(usex truetype)
+		-DENABLE_IMAGE_XPM=$(usex xpm)
+		-DENABLE_PANGO=$(usex pango)
+		-DTESTS=$(usex test)
+	)
```

Comment:
I see new configure options in 0.4.1,
```
+option(ENABLE_CURL "include support for CURL HTTP client" ON)
+option(ENABLE_LIBUDEV "include support for libudev" ON)
```
What does curl support do? 

Note that udev support must depend on **virtual/libudev** for an implementation.

Reply:
None of these are mentioned in release notes. I guess curl might be necessary for themes loading. I will make necessary changes as soon as I find out why these are included.